closed/src/java.base/share/classes/jdk/crypto/jniprovider/NativeCrypto.java

```diff
@@ -220,7 +220,7 @@ public final native int DigestComputeAndReset(long context,
 
     public final native void DigestReset(long context);
 
-    /* Native interfaces shared by CBC and ChaCha20 */
+    /* Native interfaces shared by CBC, GCM and ChaCha20 */
```

Comment:
Please list ciphers alphabetically and finish the sentence with a period.
```java
    /* Native interfaces shared by CBC, ChaCha20 and GCM. */
```

Reply:
Fixed.